central-portal/src/main/kotlin/com/vanniktech/maven/publish/portal/SonatypeCentralPortal.kt

```diff
@@ -97,15 +96,6 @@ class SonatypeCentralPortal(
     return destinationFile
   }
 
-  private fun browseDeployments(input: FileRequest): List<RepositoryActivity>? {
-    val listResponse = service.browseDeployments(input).execute()
-    if (listResponse.isSuccessful) {
-      return listResponse.body()
-    } else {
-      throw IOException("Failed to browse deployments for: $input msg: ${listResponse.errorBody()?.string()}")
-    }
-  }
```

Comment:
Removed this one since it made the build fail and I couldn't find it here https://central.sonatype.org/publish/publish-portal-api/#publish-or-drop-the-deployment